src/lib/viewers/image/ImageViewer.js

```diff
@@ -507,6 +511,23 @@ class ImageViewer extends ImageBaseViewer {
         this.annotator.toggleAnnotationMode(nextMode);
         this.processAnnotationModeChange(nextMode);
     }
+
+    handleFSMChange = () => {
+        this.updateDiscoverabilityResinTag();
+    };
+
+    updateDiscoverabilityResinTag() {
+        if (!this.containerEl) {
+            return;
+        }
+
+        const isDiscoverable = this.annotationControlsFSM.getState() === AnnotationState.REGION_TEMP;
+        const isUsingDiscoverability = this.options.enableAnnotationsImageDiscoverability && isDiscoverable;
+
+        // For tracking purposes, set property to true when the annotation controls are in a state
+        // in which the default discoverability experience is enabled
+        this.containerEl.setAttribute('data-resin-discoverability', isUsingDiscoverability);
```

Comment:
Should this string be made into a constant since it's now used in two separate viewers?